sdk/src/main/java/com/mapbox/maps/MapboxMap.kt

```diff
@@ -1640,6 +1640,32 @@ class MapboxMap :
     nativeMap.setMemoryBudget(memoryBudget)
   }
 
+  /**
+   * Sets whether multiple copies of the world will be rendered side by side beyond -180 and 180 degrees longitude.
+   * If disabled, when the map is zoomed out far enough that a single representation of the world
+   * does not fill the map's entire container, there will be blank space beyond 180 and -180 degrees longitude.
+   * In this case, features that cross 180 and -180 degrees longitude will be cut in two
+   * (with one portion on the right edge of the map and the other on the left edge of the map) at every zoom level.
+   *
+   * By default, renderWorldCopies is set to `true`.
+   *
+   * @param renderWorldCopies The `boolean` value defining whether rendering world copies is going to be enabled or not.
+   */
+  fun setRenderWorldCopies(renderWorldCopies: Boolean) {
```

Comment:
```suggestion
  fun setRenderWorldCopies(renderWorldCopies: Boolean = true) {
```

Reply:
@pengdev we seem not to use parameters by default in other places in MapboxMap, why do you think is it needed here?

Reply:
We have default values for this setting, isn't it, indicating here would be easier to find, even in generated API docs.

Reply:
We discussed this in a private message
Let's leave it as it is.